cmd/kyma/install/cmd.go

```diff
@@ -205,7 +206,9 @@ func (cmd *command) Run() error {
 }
 
 func (cmd *command) validateFlags() error {
-	if cmd.opts.Local {
+	if cmd.opts.Source == "local" {
```

Comment:
I have some points regarding the flag parsing:
- I do not see the option latest? how do I install master now?
- What happens if i pass a version that does not follow semantic versioning (e.g. 123456)?
- What happens if I pass a docker image without a tag so there is no `:`? Basing selection on that is very unstable.
- The whole nested conditionals are not robust and not readable, I think we should have something more clean:
```go
switch o.Source {
    case "local":
        <do local setup>
    case "latest":
        <do master setup>
    case isSemVer(o.Source):
        <do release setup>
    case isDockerImage(o.Source):
        <do custom image setup>
    default:
        return errors.New("Invalid installation source")
}
```